### DISCUSSION TITLE
Trying to list my service on OBS

### DISCUSSION THREAD

**Discussion by GameSeeTv** [obsproject/obs-studio#7908] [General]
Hi, I am trying to list our streaming service on OBS and was following the steps given on https://github.com/obsproject/obs-studio/wiki/Service-Submission-Guidelines but when I try to check it on my local system after changing both the files mentioned the application does not reflect my changes.

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!
